package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Please drop the changes in that file, I'll update it for a new release since we need to get another one out anyway and need to sync this with the release notes in the README. But again, I'll take care of this after the merge :+1: